cmd/tree-walk-pool.go

```diff
@@ -25,7 +25,9 @@ import (
 
 // Global lookup timeout.
 const (
-	globalLookupTimeout = time.Minute * 30 // 30minutes.
+	globalLookupTimeout    = time.Minute * 30 // 30minutes.
+	treeWalkEntryLimit     = 50
```

Comment:
Does this mean that we only cache 50 parallel listing in maximum ? isn't this a bit low ?

Reply:
It is 50 *ongoing* with different params.  So we can have 50*4 in theory. Each being ~1.1MB, total max ~220MB.

I think that is a very reasonable number. The worst thing that will happen is that some of them will be a bit slow.

Reply:
If it is too slow, then don't hammer the servers.